backend/app/services/knowledge/permission_webhook.py

```
@@ -0,0 +1,192 @@
+def _get_db_session(db_url: str) -> Session:
+    """Create a new database session for background tasks."""
+    engine = create_engine(db_url.replace("+asyncmy", "+pymysql"))
+    SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+    return SessionLocal()
+
+
+def send_permission_request_notification(
+    db_url: str,
+    permission_id: int,
+    kb_id: int,
+    applicant_id: int,
```

Ruff (0.14.14)

[warning] Unused function argument: `applicant_id`

(ARG001)